### PR TITLE
Adding Solutions

### PR DESCRIPTION
**Pull request by Pr0teus**
Added support for Solutions API

### CODE REVIEW

**Comment by sjkingo** [comment]
@Pr0teus this seems to be failing the Python 2.7 tests, which we do (currently) still support. Could you have a look please?

File: freshdesk/v2/models.py [hunk 1]
```diff
@@ -123,5 +123,37 @@ def __str__(self):
...
+        return '<Category \'{}\'>'.format(self.name)        
+
+class Folder(FreshdeskModel):
+    def __str__(self):
+        return self.name
+
+    def __repr__(self):
+        return '<Folder \'{}\'>'.format(self.name)
+
+    #@property
```
Comment (sjkingo) [inline freshdesk/v2/models.py]: Is this commented out code needed? If not, could you please remove?

File: freshdesk/v2/models.py [hunk 2]
```diff
@@ -123,5 +123,37 @@ def __str__(self):
...
+ 
+
+class Article(FreshdeskModel):
+    def __str__(self):
+        return self.title
+
+    def __repr__(self):
+        return '<Article \'{}\'>'.format(self.title)
+
+    #@property
```
Comment (sjkingo) [inline freshdesk/v2/models.py]: This too :-)

**Comment by sjkingo** [comment]
I just committed a couple of fix ups, otherwise apart from the comments above it all looks good!

File: freshdesk/v2/models.py (hunk 2 above)
Comment (Pr0teus) [reply freshdesk/v2/models.py]: I`ll create a Python2.7 environment to test and fix =)

Comment (sjkingo) [reply freshdesk/v2/models.py]: Great, thanks. I'll leave this out of current release but can do another point release when it's ready.

**Comment by sjkingo** [comment]
@Pr0teus do you still require this? If so would you able able to fix up the PR so I can get it merged? Thanks

**Comment by sjkingo** [comment]
Hi, there seems to be a lot of extra files committed that shouldn't be, would you be able to remove and force push, and I can review then.
Thanks!

**Comment by sjkingo** [comment]
Closing this, please open a new PR if you are able as this work looks good, but just needs some cleanup first :+1:

**Pull request closed without merging**